packages/abi-gen/tslint.json

```diff
@@ -1,6 +1,6 @@
 {
     "extends": ["@0x/tslint-config"],
     "linterOptions": {
-        "exclude": ["./test/generated-test/generated-artifacts/**/*", "**/lib/**/*"]
+        "exclude": ["./test-cli/fixtures/**/*", "**/lib/**/*"]
```

Comment:
so are we running linters against the generated wrappers? (i don't see them excluded here).  that would be great!!

Reply:
yes we are!